common/test/acceptance/pages/studio/settings_graders.py

```diff
@@ -14,3 +14,91 @@ class GradingPage(CoursePage):
 
     def is_browser_on_page(self):
         return self.q(css='body.grading').present
+
+    def letter_grade(self, selector):
+        """
+        Returns: first letter of grade range on grading page
+        Example: if there are no manually added grades it would
+        return Pass, if a grade is added it will return 'A'
+        """
+        return self.q(css=selector)[0].text
+
+    def click_new_grade_button(self):
```

Comment:
Function's name can be made changed to something which reflects that it not only is clicking the new grade button but also saving the changes. Simply we can make it 'add_new_grade'